tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+import logging
+
+logging.getLogger("matplotlib").setLevel(logging.WARNING)
```

Comment:
matplotlib is known to spam logs, thus we set the logging level to warning.

Reply:
sounds good, might be great to have the same comment in the source code so people do not get rid of it, lacking context